TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/AccumulationCallbacks.java

```diff
@@ -18,11 +18,10 @@ void onTrafficStreamsExpired(RequestResponsePacketPair.ReconstructionStatus stat
                                  @NonNull IReplayContexts.IChannelKeyContext ctx,
                                  @NonNull List<ITrafficStreamKey> trafficStreamKeysBeingHeld);
 
-    void onConnectionClose(int channelInteractionNumber,
-                           @NonNull IReplayContexts.IChannelKeyContext ctx,
+    void onConnectionClose(int channelInteractionNum,
+                           IReplayContexts.@NonNull IChannelKeyContext ctx, int channelSessionNumber,
```

Comment:
Did you intend to keep this `ReplayContexts.@NonNull IChannelKeyContext ctx`?

Reply:
That's a very strange way to annotate.  I've changed it to `@NonNull ReplayContexts.IChannelKeyContext`.